install.ps1

```diff
@@ -0,0 +1,95 @@
+#Requires -Version 7.0
+
+param (
+    [Parameter(Mandatory = $false)] [System.Management.Automation.SemanticVersion]$DaggerVersion,
+    [Parameter(Mandatory = $false)] [string]$InstallPath = $env:HOMEPATH + '\dagger',
+
+    [Parameter(Mandatory = $false)] [System.Boolean]$InteractiveInstall = $false
+)
+
+@"
+---------------------------------------------------------------------------------
+Author: Alessandro Festa
+Co Author: Brittan DeYoung
+Dagger Installation Utility for Windows users
+---------------------------------------------------------------------------------
+"@
```

Comment:
I'm not sure about keeping this, none of our other install scripts or tools have author names like this.

Reply:
I added a commit to convert this author block to a comment instead of it being output on every execution.